python/ray/worker.py

```diff
@@ -73,6 +73,9 @@
 DEFAULT_ACTOR_METHOD_CPUS_SPECIFIED_CASE = 0
 DEFAULT_ACTOR_CREATION_CPUS_SPECIFIED_CASE = 1
 
+# Default logger.
+logger = ray.logger.default_logger
```

Comment:
this is set at import time, which means that if we do `set_logger` later at runtime, this `logger` won't be updated.
(Just a note, if we have one logger per file, this won't be a problem.)